spec/controllers/hyrax/collections_controller_spec.rb

```diff
@@ -66,7 +66,8 @@
 
       context "without a referer" do
         it "sets breadcrumbs" do
-          expect(controller).to receive(:add_breadcrumb).with(I18n.t('hyrax.dashboard.title'), Hyrax::Engine.routes.url_helpers.dashboard_path(locale: 'en'))
+          expect(controller).to
```

Comment:
You can't have a line break here.  The ruby interpreter doesn't understand how to parse this correctly.

try this instead: 
```ruby
expect(controller).to receive(:add_breadcrumb)
  .with(...)
```